distrib/compose/docker-compose.mail.yaml

```diff
@@ -0,0 +1,15 @@
+# If you don't have a mailserver you can use the provided mailslurper
```

Comment:
We should name this to `docker-compose.mail-test.yaml` to reflect this is only for testing or local purposes, no? And ofc we should extend this in the description in this file.